src/components/Textarea/Textarea.css

```diff
@@ -0,0 +1,23 @@
+.textarea {
```

Comment:
what is the purpose of having transpired SCSS in repo?

Reply:
Sorry, my .gitignore rule was not good. My mistake.